app-modules/application/src/Observers/ApplicationSubmissionObserver.php

```diff
@@ -59,7 +59,7 @@ public function created(ApplicationSubmission $submission): void
         );
 
         if (! is_null($submission->author)) {
-            Cache::tags('application-submission-count')
+            Cache::tags('{{application-submission-count}}')
```

Comment:
This should only have one set of `{}` around this.

In the example you are copying from, it only has the double `{}` because it is within a string with a variable in it and String interpolation.